google/cloud/storage/client.h

```diff
@@ -332,12 +332,16 @@ class Client {
   /**
    * Fetches the list of buckets for the default project.
    *
-   * The default project is required to be configured in the `ClientOptions`
-   * used to construct this object. If the application does not set the project
-   * id in the `ClientOptions`, the value of the `GOOGLE_CLOUD_PROJECT` is
-   * used. If neither the environment variable is set, nor a value is set
-   * explicitly by the application, the returned `ListBucketsReader` will
-   * return an error status when used.
+   * This function will return an error if it cannot determine the "default"
+   * project. The function will use:
```

Comment:
> "The function will use:"

Perhaps something more like "The default project is found by looking, in order, for:".  Then you don't have to repeat the "If not such value is found:" each time.

Reply:
Done, thanks.